### PR TITLE
PWGGA/GammaConv added documentation file for the gammaconversion group

### PR DESCRIPTION
**Pull request by mikesas**
This commit is to add a file for the doxygen documentation of AliPhysics, for our PCG software to show up on the alidoc website.

### CODE REVIEW

**Comment by dmuhlhei** [comment]
+1

**Comment by dmuhlhei** [comment]
Hi all, is there an issue with this PR / what is the reason not to merge it?

File: doxygen/Doxyfile.in [hunk 1]
```diff
@@ -787,6 +787,7 @@ INPUT                  = @CMAKE_SOURCE_DIR@/doxygen \
                          @CMAKE_SOURCE_DIR@/PWGGA/CaloTrackCorrelations/macros \
                          @CMAKE_SOURCE_DIR@/PWGGA/CaloTrackCorrelations/macros/QA \
                          @CMAKE_SOURCE_DIR@/PWGGA/GammaConv \
+			 @CMAKE_SOURCE_DIR@/PWGGA/GammaConv/documentation \
```
Comment (dberzano) [inline doxygen/Doxyfile.in]: Please use spaces, not tabs.

**Comment by dmuhlhei** [comment]
+1

**Comment by dmuhlhei** [comment]
+1

**Comment by dberzano** [comment]
+1

**Pull request merged**